source/views/index.html.haml

```diff
@@ -1,6 +1,17 @@
 ---
 title: GradientDaze
 ---
+.explainer.animated.zoomIn
+  %header
+    =inline_svg("G-Logotype.svg", class: "g-logotype")
```

Comment:
This might be overkill, but could be fun to try in a v2: https://css-tricks.com/conical-gradients-css/